README.md

```diff
@@ -70,6 +70,10 @@
 
 * [Hugo blog](https://blog.mrtrustor.net/post/making-this-blog-with-cloud-run/)
 * [Apache NiFi workflows](https://medium.com/@pierre.villard/deploying-apache-nifi-workflows-on-google-cloud-run-8c0c988354f1)
+* Samples and tutorials with the [Micronaut](https://micronaut.io/) framework
```

Comment:
There is already a section about Java that has a link to a micronaut sample. Would you mind adding these at this location?

Reply:
Updated.